apps/sms/index.html

```diff
@@ -84,6 +84,9 @@
     <script defer src="views/shared/js/settings.js"></script>
     <script defer src="services/js/message_manager.js"></script>
     <script defer src="views/shared/js/app.js"></script>
+    <script defer src="services/activity/threads.js"></script>
```

Comment:
and name :) "threads" is not good for us, I think Steve chose "bridge" which fits more our app.

Maybe we should have a simple script to update it (using browserify etc); the script shoud better be js based so that we could call it during the build in build.js.